ci-operator/config/red-hat-data-services/notebooks/red-hat-data-services-notebooks-release-2023a.yaml

```diff
@@ -117,6 +117,25 @@ images:
   dockerfile_path: Dockerfile
   from: base-anaconda-python-3.8
   to: jupyter-datascience-anaconda-python-3.8
+- context_dir: base/c9s-python-3.9
```

Comment:
lets drop the release branches for now, 2023a and 2023b
once our testing is done, lets add them back